UndertaleModLib/Compiler/Parser.cs

```diff
@@ -668,6 +690,9 @@ private static Statement ParseFunction(CompileContext context)
                 {
                     expressionMode = false;
                     Statement s = remainingStageOne.Dequeue();
+                    if (s.Text.StartsWith("___struct___")) {
```

Comment:
Is it possible for any real GM games out in the wild to have functions that start with `__struct__`? If not, you may wanna slap a todo onto this.

Reply:
GM does seem to let you compile functions that start with `___struct___`, so maybe (though unlikely).